content/schema/8.md

```diff
@@ -6,6 +6,10 @@ title = "Predicate Query"
 weight = 8
 +++
 
+{{% notice "note" %}}
+  `_predicate_` doesn't work on Dgraph 1.1 but we plan to add a new function to do the same job.
```

Comment:
maybe remove this. It's still not sure whether we will add an equivalent feature. Maybe say for now users can query the types (just like the schema) to get the fields that belong to that type.